source/collections/Vector2D.ooc

```diff
@@ -38,6 +40,58 @@ Vector2D: class <T> {
 	_elementPosition: func (row, column: Int, columnCount := this columnCapacity) -> Int {
```

Comment:
The confusion is justified. `rowCapacity` here was referring to the capacity of a single row, i.e. the column count. The naming was inherited from the `Vector` class. I will switch to using the term count instead and make changes in the code accordingly.